content/en/docs/v3.4/op-guide/configuration.md

```diff
@@ -17,6 +17,8 @@ The [official etcd ports][iana-ports] are 2379 for client requests and 2380 for
 
 To start etcd automatically using custom settings at startup in Linux, using a [systemd][systemd-intro] unit is highly recommended.
 
+The list of flags provided below is for example purposes. It may not be up-to-date. For the latest available flags, run `etcd --help` or refer to the [etcd help][].
```

Comment:
We still need to make sure the doc is in sync with the help.go, but there maybe some delay. So I think the flags provided below are still supposed to be a full list instead of examples. WDYT? 

```suggestion
The list of flags provided below may not be up-to-date due to ongoing development changes. For the latest available flags, run `etcd --help` or refer to the [etcd help][].
```

Reply:
@ahrtr +1 I like the tweak here. Will update. Thanks!